Update deep link docs due to the flutter_deeplinking_enabled flag default value change. (#11461)
flutter/website: src/content/ui/navigation/deep-linking.md

If you have written a plugin to handle deep links, as described in
[Deep Links and Flutter applications][plugin-linking]
(a free article on Medium),

> You should opt out the default flutter deep links handler by adding
> `FlutterDeepLinkingEnabled` to false in `Info.plist` or
> `flutter_deeplinking_enabled` to false in `AndroidManifest.xml`, respectively.

Reviewer: ```suggestion
you should opt out the Flutter's default deep link handler.
To do this, set `FlutterDeepLinkingEnabled` to false in `Info.plist` _or_
`flutter_deeplinking_enabled` to false in `AndroidManifest.xml`.
```

Just double checking. You don't need to do both? It's one or the other?

Author: One for iOS and one for Android,  

You don't need to do both? It's one or the other? >>It depends on if you develop on both platform or just one.
